tests/console/soundtouch.pm

```diff
@@ -18,9 +18,23 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
+use version_utils 'is_sle';
 
 sub run {
     select_console 'root-console';
+    # development module needed for dependencies, released products are tested with sdk module
+    if (get_var('BETA')) {
```

Comment:
I find this approach dangerous  . test modules should not play with registration . they should run against properly prepared image which has setup. this can lead to several problems : 
1. not clear test status . failures in register/unregister which happening pretty often will confuse test reviewer if problems is actually with soundtouch or with registration 
2. registration process changing something **outside** SUT so when something broken there it will break not only soundtouch module but everything which goes after it and **all**  other jobs which using same image 

@okurz , @rwx788 , @Soulofdestiny - WDYT ?

Reply:
Yes, all this shit is only because of QA, QAM images are prepared with all needed modules.

Reply:
Well, I have seen multiple modules "playing" with registration already and it is true that this is also leading to problems. For SLE15+ commonly `add_suseconnect_product` is called instead of using a bare repo directly but as mentioned, this is bringing the problem of not knowing afterwards if we can rely on registration. I would not call it "shit" what is necessary here though. It just very much depends on what is the test target. Obviously here the goal is to just test "soundtouch", not try out if the "SDK" is usable. So actually any way goes, I am fine with either way, the current way looks ok.

Reply:
Maybe not shit, but mess for sure, maybe consider to exclude this packages or create new group which requires 'sdk'